src/OpenLoco/src/Drawing/SoftwareDrawingContext.cpp

```diff
@@ -1472,9 +1473,282 @@ namespace OpenLoco::Drawing
             call(0x0045196C, regs);
         }
 
+        // 0x00451582
+        static int16_t drawStringMaxChars(RenderTarget& rt, Ui::Point origin, const AdvancedColour colour, uint8_t* str, const int16_t numCharsRemaining)
+        {
+            // This function has been somewhat simplified removing unreachable parts
+            if (!colour.isFE())
+            {
+                assert(false);
+                return numCharsRemaining;
+            }
+            int16_t numChars = numCharsRemaining;
+            Ui::Point pos = origin;
+            while (true)
+            {
+                if (numChars == 0)
+                {
+                    break;
+                }
+                // When offscreen in y dimension don't draw text
+                // In original this check only performed if pos.y updated instead of every loop
+                bool offscreen = true;
+                if (pos.y + 19 > rt.y)
```

Comment:
what is 19, the height of the ticker box?

Reply:
this is copied from other code but I'm pretty sure 19 is the largest in pixels that a character could possibly be.